### DISCUSSION TITLE
How can I get belongsTo relationship data in the relation-manager

### DISCUSSION THREAD

**Discussion by mra-narola** [filamentphp/filament#6017] [Help] [answered]
Using the Filament, I created the Country, State, and City modules. and I have an Employee Module which belongs to City.

I want to display Country, State, and City, wise employees.
In the City Module, I am able to display employees, but not able to get how can I show the Country and State Module.

So my table structure will be like...

```
employees
- id
- name
- email
- address
- city_id
---------------------------------
countries
- id
- country_code
- name
---------------------------------
states
- id
- country_id
- name
---------------------------------
cities
- id
- state_id
- name
```
```
Relationship

class Employee extends Model
{
    // model code...
    public function city(): BelongsTo
    {
            return $this->belongsTo(City::class);
    }
}


class City extends Model
{
    public function employees(): HasMany
    {
        return $this->hasMany(Employee::class);
    }
}
```

Is there any way, where we can use inverse/belongsTo relationship data in the relation-manager, and how I archive this?

Any suggestions or help?
Thanks

**Comment by danharrin** [comment, marked as answer]
Relation managers are for managing multiple records in a table format. BelongsTo and HasOne do not fit this pattern since they deal with 1 related record.

If you want to edit a BelongsTo from the edit page of its parent resource, you could use a Select or https://filamentphp.com/docs/2.x/forms/layout#saving-data-to-relationships

Or you could build a widget that has the form builder embedded

**Reply by mra-narola** [reply]
@danharrin thanks for your feedback mate!...
Make sense.

In the future, is there any chance to come up with this pattern? :)

**Reply by danharrin** [reply]
The pattern is already possible in other ways, as I said. So not sure if we need another way on top of those
